data/pages-content/pyladies-workshop.md

```diff
@@ -0,0 +1,25 @@
+# Beginners' Day - PyLadies Snake Workshop
+## About PyLadies
+PyLadies is an international mentorship group with focus on helping more women become active participants and leaders in the Python open-source community.
+Our mission is to promote, educate and advance a diverse Python community through outreach, education, conferences, events and social gatherings.
+
+## About the Workshop 
+Are you a member of an under-represented group interested in programming but unsure where to begin? Don't worry, we have the perfect workshop for you, designed to be welcoming and accessible even if you don't have any prior programming experience!
+
+With the support of experienced mentors, you'll create your very own version of the classic Snake video game, reminiscent of those on older Nokia mobile phones. You'll get hands-on experience writing code, learning fundamental programming concepts, and feeling the satisfaction of a computer finally obeying your commands.
+
+If the project seems impractical, fear not! The main goal is to teach you basic programming principles that you can later apply to various fields, such as web application development, data analysis, machine learning, robotics, or system tool design.
+
+This workshop is specifically tailored for total beginners. There's no need for prior programming knowledge; all you need is a computer and basic skills to use it (e.g., opening/saving files, installing programs).
+
+The workshop will be conducted in English and is completely free. However, space is limited, so be sure to register through the pyladies.cz website.
+
+If this introduction to programming sparks your interest, you can further explore the world of coding with one of the PyLadies autumn beginner courses, starting shortly after the workshop.
```

Comment:
I'd skip this sentence entirely or underline the courses are in Czech, so that we don't create false expectations.

Reply:
I've removed this sentence. I've also added the link you've sent us.